### PR TITLE
Add middle button support

### PR DESCRIPTION
**Pull request by jbensmann**

### CODE REVIEW

File: mouse_test.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package uinput
+package main
```
Comment (bendahl) [inline mouse_test.go:1]: What's the reason behind this change of package? Is this intended or the result of an automatic refactoring performed by the IDE? If this change is intentional, it would be interesting to understand the reasoning behind it. Otherwise, it would be strongly preferred to keep this consistent with the rest of the package.

**Comment by bendahl** [comment]
Hi @jbensmann.

first of all, thank you very much for your contriubtion. I would very much like to include this change. There is only one open question for me right now regarding the package of the test code. Can you please shed some light on this. Otherwise the change is looking good. Thanks again for taking your time to improve this tool!

Cheers,
Ben

**Comment by jbensmann** [comment]
Hi, changing the package was not intended, it must have happened during some refactoring, I have changed it back again.

File: mouse_test.go [hunk 2]
```diff
@@ -54,9 +54,14 @@ func TestBasicMouseMoves(t *testing.T) {
...
 
+	err = relDev.MiddleClick()
+	if err != nil {
+		t.Fatalf("Failed to perform middle click. Last error was: %s\n", err)
+	}
+
 	err = relDev.LeftPress()
 	if err != nil {
-		t.Fatalf("Failed to perform left key press. Last error was: %s\n", err)
+		t.Fatalf("Failed to perform left key isPress. Last error was: %s\n", err)
```
Comment (djsavvy) [inline mouse_test.go:64]: Quick question — why the change from `press` to `isPress` here and elsewhere?

Comment (jbensmann) [reply mouse_test.go]: This must also have happened during some refactoring, I have changed it back. Sorry, I should have checked the diff more carefully before creating a pull request :)

**Review by djsavvy — APPROVED** [review]
Not the package author, but just an interested consumer. This patch set looks good to me!

File: mouse_test.go (hunk 2 above)
Comment (djsavvy) [reply mouse_test.go]: Haha. Thanks!

**Comment by bendahl** [comment]
Hi guys,

great work! I am happy to see so much interested in this project.

@jbensmann: Thanks for the clarification and prompt change.
@djsavvy: Thanks for taking the time to review this change. I agree that it's looking good now. I will merge the change.

Cheers,
Ben
